pkg/vm/engine/tae/mergesort/uuids/func.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Matrix Origin
```

Comment:
Did you feel the pain!    TAE folks need to fix this ...

Reply:
Yes, I see that the code of the sorting algorithm can't be reused and needs to be reconstructed